src/main/kotlin/no/nav/syfo/util/StringUtil.kt

```diff
@@ -0,0 +1,8 @@
+package no.nav.syfo.util
+
+import java.util.*
+
+fun String.lowerCapitalize() =
+    this.lowercase(Locale.getDefault()).replaceFirstChar {
+        if (it.isLowerCase()) it.titlecase(Locale.getDefault()) else it.toString()
+    }
```

Comment:
Denne kan vel egentlig hete bare `capitalize`, siden det betyr stor forbokstav og resten små.
Og kan man ikke bare erstatte den første karakteren med stor bokstav uansett? Det er vel ikke nødvendig å sjekke om den er stor først.

```
A capitalised word = Word
An upper case word = WORD
A lower case word = word 
A capitalized sentence = Capitalized sentence
First letter only = Ordfabrikken AS
```

Reply:
Fint det, endret til `String.capitalize()`.

Reply:
Jeg tenkte ikke på at capitalize eksisterer, men så lenge kotlin er fornøyd er det jo greit. Kotlin sin innebygde metode er visst deprecated til fordel for `replaceFirstChar` 🤷‍♂️

Reply:
Ja, men ser ut som om den funksjonen bare endrer først bokstav, det jeg ønsker er at alle bokstaver vil små utenom evt først bokstav som blir capitalized.

Ser ut som om Kotlin sin capitalize gir følgende:
bOOK.capitalize() -> BOOK.
men jeg ønsker:
bOOK.func() -> Book.